doc/changelog/07-commands-and-options/12493-declare-constr-hint.rst

```diff
@@ -0,0 +1,7 @@
+- **Changed:**
+  Declaring a hint which is not a global reference now generates
+  a constant on the fly. This behaviour can be tweaked by
```

Comment:
```suggestion
  a constant on the fly. This behavior can be tweaked by
```

The doc prefers the American spelling by a wide margin.  And "colour" not at all.  Mixing American and British spelling in one document is a bit weird.

Reply:
AFAIR we actually *require* US English in the Coq documentation, but I can't help writing some briticisms from time to time.